### ISSUE TITLE
Alternative strip: llvm-strip via cargo-binutils

### ISSUE THREAD

**Issue by ZeroErrors** [johnthagen/min-sized-rust#15]
Consider adding [`cargo-binutils`](https://github.com/rust-embedded/cargo-binutils) to the README. It provides easy to use cargo commands for a few llvm tools that can be installed via rustup. These tools work for quite a few platforms including windows and is easy to get setup if you use `rustup` for your rust toolchain.
```
$ rustup component add llvm-tools-preview
$ cargo install cargo-binutils
$ cargo strip
```

The `cargo-binutils` cargo commands automatically handle building the project if needed as well as finding and passing the target binary path to the tool. They also support the majority of the command line options that `cargo build` does to make the tools as intuitive as using cargo itself.

Also provides access to `llvm-size` via `cargo size`.

https://github.com/rust-embedded/cargo-binutils

**Disclaimer:** I recently did a little bit of work on `cargo-binutils` to make it easier to use and more closely match the built in cargo commands, and the README is a little out of date because of that.

**Comment by yoshuawuyts** [comment]
This is available directly from Cargo on nightly as well since https://github.com/rust-lang/cargo/pull/8246. I believe the syntax to run it is:

```sh
$ cargo +nightly build -Z strip=symbols
```

**Comment by johnthagen** [comment]
@yoshuawuyts It'd like to add the reference to the nightly flag, as it's pretty simple to use (and hopefully stabilized eventually).

Do you happen to know which nightly was the earliest it's available in?

**Comment by yoshuawuyts** [comment]
@johnthagen this was merged on may 27th, so that would be the 1.45 nightlies. I tried finding the exact nigthly on forge but couldn't find it -- mentioning something like: _"Available starting 1.45.0-nightly (2020-05-28)"_ should be good enough for most I think.

**Comment by johnthagen** [comment]
Fixed in https://github.com/johnthagen/min-sized-rust/commit/1c13bc629f7df3e55b3e3fb712234c033e6dceab.

Thanks @ZeroErrors and @yoshuawuyts for the suggestion!
